README.md

```diff
@@ -56,6 +62,88 @@ If you have a Snyk token, add that as an environment variable:
 - Click on Manage Jenkins -> System, scroll down to Global properties/Environment Variables.
 - Set the variable name to _EXHORT_SNYK_TOKEN_, and copy-and-paste your Snyk token into the value field.
 
+
+#### General Configuration
+ Click on Manage Jenkins -> System, scroll down to Global properties/Environment Variables, And there you can set the following settings:
```

Comment:
Suggested rewrite:

`Click on Manage Jenkins -> System, scroll down to Global properties/Environment Variables, And there you can set the following settings:` -->

`Click <em>Manage Jenkins</em>.  Click <em>System</em>, and scroll down to <em>Global properties/Environment Variables</em>. Here you can configure the following settings:`

Reply:
Done.